### PR TITLE
Initial attempt at Sky brightness Object

### PR DESCRIPTION
**Pull request by cbard**
This is a simple Sky brightness class using data taken from Neugent & Massey (2010) for Kitt Peak. 

I had issues with the procedure for converting the data (in mag/arcsec^2) into flux (ergs/sec/cm^2/angstrom/arcsec^2), so there is likely a better way to do this.

### CODE REVIEW

**Comment by cbard** [comment]
After contacting Phil Massey, I clarified that the data was indeed in AB magnitudes. This makes it straightforward to convert to f_nu and f_lambda.

File: m4opt/models/background/skybright.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+    References
+    ----------
+    .. [1] Neugent, K. and Massey, P., 2010, "The Spectrum of the Night Sky
+           Over Kitt Peak: Changes Over Two Decades". PASP, 122, 1246-1253
+           doi:10.1086/656425
+    """
+
+    if option.lower() not in kpno_sky_tables.keys():
+        raise AttributeError("option must be one of {0}".format(
```
Comment (lpsinger) [inline m4opt/models/background/skybright.py:47]: Should be a `ValueError` ("Raised when an operation or function receives an argument that has the right type but an inappropriate value")

File: m4opt/models/background/skybright.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    not yet include effect of lunar phase.
+
+    References
+    ----------
+    .. [1] Neugent, K. and Massey, P., 2010, "The Spectrum of the Night Sky
+           Over Kitt Peak: Changes Over Two Decades". PASP, 122, 1246-1253
+           doi:10.1086/656425
+    """
+
+    if option.lower() not in kpno_sky_tables.keys():
```
Comment (lpsinger) [inline m4opt/models/background/skybright.py:46]: More Pythonic ("easier to ask for forgiveness than permission"):

```python
key = option.lower()
try:
    filename = keno_sky_table[key]
except KeyError:
    raise ...
```

**Review by lpsinger — CHANGES_REQUESTED** [review]
Very nice! Suggesting two minor changes.

File: m4opt/models/background/__init__.py [hunk 3]
```diff
@@ -1,7 +1,8 @@
 from .core import Background
 from .zodiacal import ZodiacalBackground
+from .skybright import SkyBrightness
 
-__all__ = ('ZodiacalBackground',)
+__all__ = ('ZodiacalBackground', 'SkyBrightness',)
```
Comment (lpsinger) [inline m4opt/models/background/__init__.py:5]: Do we want to call this `SkyBackground`, for consistency with the other classes in this module?

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: m4opt/models/background/__init__.py (hunk 3 above)
Comment (cbard) [reply m4opt/models/background/__init__.py:5]: Yes, we should do that.

**Pull request merged**